src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -414,6 +414,16 @@ object ItemFeatures {
                 }"
             })
         }
+
+        if (Skytils.config.showItemQuality && extraAttr != null) {
+            val boost = extraAttr.getInteger("baseStatBoostPercentage")
+            val tier = extraAttr.getInteger("item_tier")
+
+            if (boost > 0 && tier > 0) {
+                event.toolTip.addAll(listOf("§6Base Stat Boost: §b$boost%", "§6Item Tier: §b$tier"))
```

Comment:
I don't think "item_tier" works well enough. Doesn't work well with master mode (Example, item_tier in Master Floor 3 is 6, same as floor 6. Floor 7 is 7, same as Master Floor 4.

Instead, dungeon_skill_req is the same as the requirement to enter the floor it dropped from and it would work for differentiating between normal floors and master mode. 
Also instead of adding "Item Tier:" and the tier i think you can just put that between parenthesis after the stat boost

Reply:
I mean knowing whether it came from F7 or M4 doesn't change anything does it? Since the boost both of them give is the same is it not?

Reply:
> I mean knowing whether it came from F7 or M4 doesn't change anything does it? Since the boost both of them give is the same is it not?

Not sure (since you would need 2 armor pieces with same stat boost to check), but imo its better to do it that way if its going to support master mode anyways, since it avoids confusing hypixel values like "item_tier" 10 being M7 or 8 being M5